lib/CL/devices/vulkan/pocl-vulkan.h

```diff
@@ -0,0 +1,32 @@
+/* pocl-vulkan.h - Vulkan driver callback prototypes
+
+   Copyright (c) 2018 Michal Babej <michal.babej@tut.fi>
```

Comment:
Check the year and email. Add Tampere University also as this was done under its project funding.